src/component/EditViolationModal.jsx

```diff
@@ -24,7 +24,7 @@ const EditViolationModal = ({ isOpen, onClose, onSubmit, violationToEdit }) => {
         approvedByName: ""
     });
 
-    // Validation function
+    // Validation
     const validate = () => {
         const studentNumberPattern = /CT[0-9]{2}-[0-9]{4}/;
```

Comment:
why make student number exclusive to CT pattern?

Reply:
sir, should i remove or change the validation of the studentNumberPattern?